WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListFragment.java

```diff
@@ -159,6 +182,96 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
+            if (actionEvent instanceof ActionEvent.OpenServiceDetails) {
+                final Activity activity = getActivity();
+                if (activity instanceof PublicizeListActivity) {
+                    onServiceClick(((OpenServiceDetails) actionEvent).getService());
+                }
+            }
+        });
+    }
+
+    private void setupTwitterDeprecationContainer() {
+        final View rootView = getView();
+        if (rootView != null) {
+            final View twitterDeprecationNoticeItemContainer =
+                    rootView.findViewById(R.id.publicize_twitter_deprecation_notice_item_container);
+            if (twitterDeprecationNoticeItemContainer != null) {
+                twitterDeprecationNoticeItemContainer.setOnClickListener(view -> {
+                    mPublicizeListViewModel.onTwitterDeprecationNoticeItemClick();
+                });
+            }
+        }
+    }
+
+    private void showTwitterDeprecationNotice(final ShowTwitterDeprecationNotice uiState) {
+        final View rootView = getView();
+        if (rootView != null) {
+            final View twitterContainer = rootView.findViewById(R.id.twitter_deprecation_notice_container);
+            twitterContainer.setVisibility(View.VISIBLE);
+            final TextView title = rootView.findViewById(R.id.publicize_twitter_deprecation_notice_header_service);
+            title.setText(uiState.getTitle());
+            final TextView description =
+                    rootView.findViewById(R.id.publicize_twitter_deprecation_notice_header_description);
+            final String space = " ";
+            final String descriptionText = getString(uiState.getDescription()) + space;
+            final String findOutMoreText = getString(uiState.getFindOutMore());
+            final SpannableString spannableTitle = new SpannableString(descriptionText + findOutMoreText);
+            final int descriptionColor = description.getCurrentTextColor();
+            spannableTitle.setSpan(
+                    new ForegroundColorSpan(descriptionColor),
+                    0,
+                    descriptionText.length(),
+                    Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+            );
+            spannableTitle.setSpan(
+                    new ForegroundColorSpan(ContextCompat.getColor(getContext(), R.color.jetpack_green)),
+                    descriptionText.length() + 1,
+                    spannableTitle.length(),
+                    Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+            );
+            spannableTitle.setSpan(
+                    new ClickableSpan() {
+                        @Override public void onClick(@NonNull View view) {
+                            WPWebViewActivity.openURL(getActivity(), uiState.getFindOutMoreUrl());
+                        }
+                    },
+                    descriptionText.length(),
+                    spannableTitle.length(),
```

Comment:
❓ Question: shouldn't these indices match the ones from the span above (which color the text green)?

Reply:
Yes, thanks 👍